api/api/src/main/scala/uk/ac/wellcome/platform/api/rest/ImagesParams.scala

```diff
@@ -43,7 +43,7 @@ object MultipleImagesParams extends QueryParamsUtils {
         "page".as[Int].?,
         "pageSize".as[Int].?,
         "query".as[String].?,
-        "license".as[LicenseFilter].?,
+        "locations.license".as[LicenseFilter].?,
```

Comment:
This should be `items.locations.license` as we try to replicate the JSON path as close as possible.

[There's a small comment in this PR talking about defaulting to `id`or `value`](https://github.com/wellcomecollection/catalogue/pull/586).

Reply:
Wait, I read this wrong, I thought we were in works land, ignore.